README.md

```diff
@@ -403,6 +403,135 @@ After loading the data, run the following updates:
+- the data asset description is complete in GDB
+- the name and id of the new data asset are known
+  - default values are built by appending `-influx` to the csv data asset ID and ` in Influx` to the csv data asset Name
+  - however they should be editable by the user
+
+Metadata creation
+
+A new metadata record is created based on the source dataset 
+[create-influx-dataset.ru](queries/create-influx-dataset.ru)
+
+Uhe influx asset creation process should also 
+create the corresponding bucket in influx and 
+associate it with the organization.
+
+
+### Appending data to a influx dataset
+
+The [Extended annotated CSV](https://docs.influxdata.com/influxdb/cloud/reference/syntax/annotated-csv/extended/) syntax 
+consists of a series of lines with annotations before the actual data, 
+which is in the form of a csv file. 
+
+The annotations are of the following types:
+
+#### #datatype annotation header
+
+The `#datatype ` annotation to specifies the [line protocol element](https://docs.influxdata.com/influxdb/cloud/reference/syntax/line-protocol/#elements-of-line-protocol) a column represents. 
+The contents of the annotation depends on the semantic description of the data asset and can be fetched by the following sparql query:
+
+```sparql
+PREFIX dct: <http://purl.org/dc/terms/>
+PREFIX un: <https://dataspace.underpinproject.eu/ontology/>
+BASE <https://dataspace.underpinproject.eu/>
+PREFIX csvw: <http://www.w3.org/ns/csvw#>
+select (concat("#datatype ",(group_concat(?dt;separator=','))) as ?a) where {
+  { select ?dt where {
+    bind(<dataset/windfarm-WF1-WTG01-2020.csv> as ?dataset)
+    ?dataset dct:conformsTo ?schema .
+    ?column a csvw:Column ; ^csvw:column ?schema ; un:influxDatatype ?dt ; dct:identifier ?n.
+  } order by ?n }
+}
+```
+
+Which will produce the annotation header:
+
+```
+#datatype dateTime:1/2/2006 15:04,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,double,ignore,double,ignore,double,double,double,double,ignore,double,ignore,double,double,ignore,ignore,ignore,ignore,ignore,ignore,double,double,double,ignore,ignore,ignore,ignore,double,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,double,double,double,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,ignore,double,ignore,ignore,ignore,ignore,ignore,ignore,ignore
+```
+
+for the `<dataset/windfarm-WF1-WTG01-2020.csv>` dataset
+n.b that the order of the annotations is dependent on the explicit column index (`dct:identifier`) in the metadata

```

Comment:
- move "for ..." above the code block
- Finish sentences with full-stop
- Don't forget that markdown paragraphs need to be separated by a newline
- "n.b" -> "note that"